### DISCUSSION TITLE
你好，关于siameseFC模型

### DISCUSSION THREAD

**Discussion by byq817** [STVIR/pysot#407] [General]
您好，我是目标跟踪新手，但关注该项目很长时间了，在model Zoo中始终没有SiameseFC的模型，请问后期会更新SiameseFC嘛，还是已经存在此模型，我没有看到？谢谢

**Comment by ZhiyuanChen** [comment]
你只要使用AlexNet bakbone然后去掉新工作添加的内容（比如depthwise互相关）就是SiamFC了呀

**Reply by xyl-507** [reply]
@ZhiyuanChen 作者您好。请问一下，siamfc中没有rpn模块，需要的互相关运算应该怎么加入？
